cmake/cross-windows-x86_64.cmake

```diff
@@ -0,0 +1,28 @@
+
+set(CMAKE_SYSTEM_NAME Windows)
+set(CMAKE_SYSTEM_VERSION 10.0)
+set(CMAKE_SYSTEM_PROCESSOR x86_64)
+
+set(CMAKE_C_COMPILER x86_64-w64-mingw32-gcc)
+set(CMAKE_CXX_COMPILER x86_64-w64-mingw32-g++)
+set(CMAKE_RC_COMPILER x86_64-w64-mingw32-windres)
+set(CMAKE_FIND_ROOT_PATH /usr/x86_64-w64-mingw32)
+
+# Slang specific
+set(SLANG_LIB_TYPE STATIC)
```

Comment:
Why are we setting static library as default?

Reply:
The toolchain file is only intended for an easy way to reproduce same build, not intended for merging. You can ignore that file. This is Work in Progress anyways. I'm not expecting to merge as is. Open to ideas